src/formatDate.js

```diff
@@ -51,6 +51,53 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const newDate = [];
+  const splitDate = date.split(fromFormat[3]);
+  let newDay = '';
+  let newMonth = '';
+  let newYear = '';
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
It's okay but, what is 3 here? I mean why you use exactly 3? you can use splitDate.length for example